src/kontrol/foundry.py

```diff
@@ -835,3 +825,12 @@ def foundry_node_printer(foundry: Foundry, contract_name: str, proof: APRProof)
     if type(proof) is APRProof:
         return FoundryAPRNodePrinter(foundry, contract_name, proof)
     raise ValueError(f'Cannot build NodePrinter for proof type: {type(proof)}')
+
+
+def load_foundry(foundry_root: Path, bug_report: BugReport | None = None) -> Foundry:
+    try:
+        foundry = Foundry(foundry_root=foundry_root, bug_report=bug_report)
+    except FileNotFoundError:
+        _LOGGER.error('foundry.toml file not found. Are you running kontrol in a Foundry project?')
```

Comment:
```suggestion
        print(f'File foundry.toml not found in: {foundry_root}. Are you running kontrol in a Foundry project?', file=sys.stderr)
```

Reply:
Why `print()` and not `_LOGGER.error()`?

Reply:
`logging.error()` is for indicating suppression of an exception, `print` is for normal output: https://docs.python.org/3/howto/logging.html#when-to-use-logging

Reply:
Thank you!